07.ls_object/run_ls.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require_relative 'lib/display'
+
+params = ARGV.getopts('arl')
+paths = params['a'] ? Dir.glob('*', File::FNM_DOTMATCH) : Dir.glob('*')
+sorted_paths = params['r'] ? paths.reverse : paths
+display = Display.new(sorted_paths)
+puts params['l'] ? display.render_long_list : display.render_short_list
```

Comment:
```suggestion
puts display.render(long: params['l'])
```

のように引数でlongかどうかを指定した方がスマートかも？

Reply:
たしかに、` params['a']`や` params['r']`と違い、` params['l']`はメソッドの呼び出しに使用しているので、引数として渡すことができます。
また、キーワード引数にすると可読性が上がり、デフォルト値の設定ができることを再復習しました。